x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/IndexInputStats.java

```diff
@@ -137,14 +149,22 @@ public Releasable addIndexCacheFill() {
         };
     }
 
-    public int getNumFiles() {
+    public long getNumFiles() {
         return numFiles;
     }
 
     public long getTotalSize() {
         return totalSize;
     }
 
+    public ByteSizeValue getMinSize() {
```

Comment:
I would keep this a long in this class, or change totalSize also to use ByteSizeValue

Reply:
I keep total/min/max as long in this class and they are later converted as ByteSizeValue when aggregated in the stats as CacheIndexInputStats.